server/base/src/main/java/org/apache/accumulo/server/security/SecurityOperation.java

```diff
@@ -743,8 +743,9 @@ public void grantSystemPermission(TCredentials credentials, String user,
     }
   }
 
-  public void grantTablePermission(TCredentials c, String user, TableId tableId,
-      TablePermission permission, NamespaceId namespaceId) throws ThriftSecurityException {
+  public void grantTablePermission(TCredentials c, String user, TableId tableId, String tableName,
+      TablePermission permission, NamespaceId namespaceId)
```

Comment:
The new tableName parameter is not used in this method, is that expected?

Reply:
That is expected. It is not used in this method, but is used in the overriden method in AuditedSecurityOperation.